content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -123,6 +123,32 @@ const prisma = new PrismaClient({
 })
 ```
 
+### <inlinecode>adapter</inlinecode>
+
+Defines an instance of a driver adapter. See also [Drivers](/concepts/components/drivers) <span class="concept"></span>.
```

Comment:
Shouldn't this link to a driver adapters headline/anchor on that page?

Reply:
```suggestion
Defines an instance of a driver adapter. See also [Drivers](/concepts/components/drivers#driver-adapters) <span class="concept"></span>.
```